src/Illuminate/Validation/Validator.php

```diff
@@ -216,7 +216,7 @@ protected function explodeRules($rules)
     {
         foreach ($rules as $key => $rule) {
             if (Str::contains($key, '*')) {
-                $this->each($key, $rule);
+                $this->each($key, [$rule]);
```

Comment:
Actually if it is already an array it should be wrapped in another array, that is the fix.

So when it calls `explodeRules` again ( `explodeRules(...) >>> each(...) >>> mergeRule(...) >>> explodeRules(...)` ) and get into the `else` branch, it fails on the `is_string()` check